Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -1201,14 +1201,25 @@ public string GetSmartApiUrl(string hostname, int? port = null)
         }
 
         /// <inheritdoc/>
-        public string GetLoopbackHttpApiUrl()
+        public string GetInterfaceHttpApiUrl()
         {
-            if (NetManager.IsIP6Enabled)
+            // Published server ends with a /
+            if (!string.IsNullOrEmpty(PublishedServerUrl))
```

Comment:
This does not seem ideal. Isn't the published server url possibly a domain like https://jellyfin.example.com? If ffmpeg calls that and the router doesn't support hairpin NAT, it might not work as expected.

Reply:
Ran it through the same code to try and keep url's the same throughout.

Could change it to the following - but that would bypass and rules the user has set up.

```
var bind = NetManager.GetInternalBindAddresses().FirstOrDefault() ??
    NetManager.GetAllBindInterfaces(true).FirstOrDefault();
return GetLocalApiUrl(bind.Address.ToString(), Uri.UriSchemeHttp);
```